src/Build/PublicAPI/net/PublicAPI.Shipped.txt

```diff
@@ -1320,7 +1320,6 @@ Microsoft.Build.Experimental.ProjectCache.CacheResultType.CacheMiss = 2 -> Micro
 Microsoft.Build.Experimental.ProjectCache.CacheResultType.CacheNotApplicable = 3 -> Microsoft.Build.Experimental.ProjectCache.CacheResultType
 Microsoft.Build.Experimental.ProjectCache.CacheResultType.None = 0 -> Microsoft.Build.Experimental.ProjectCache.CacheResultType
 Microsoft.Build.Experimental.ProjectCache.PluginLoggerBase
```

Comment:
Not for this PR but we should think about moving _all_ of this Experimental stuff into Unshipped since that's basically the contract we're following for Experimental.